### PR TITLE
WIP: Rum 18632 distribute

### PR DESCRIPTION
**Pull request by khaleqi67**

### CODE REVIEW

File: TEMPLATE_BUG_REPORT.md [hunk 1]
```diff
@@ -1,36 +0,0 @@
-# Bug report
```
Comment (yghern) [inline TEMPLATE_BUG_REPORT.md:1]: Why this file is deleted? Needs updating readme if correct

File: TEMPLATE_FEATURE_REQUEST.md [hunk 2]
```diff
@@ -1,15 +0,0 @@
-# Feature request
```
Comment (yghern) [inline TEMPLATE_FEATURE_REQUEST.md:1]: Why this file is deleted? Needs updating readme if correct

File: TEMPLATE_BUG_REPORT.md (hunk 1 above)
Comment (yghern) [reply TEMPLATE_BUG_REPORT.md:1]: File moved: .github/ISSUE_TEMPLATE/bug_report.md

File: TEMPLATE_FEATURE_REQUEST.md (hunk 2 above)
Comment (yghern) [reply TEMPLATE_FEATURE_REQUEST.md:1]: File moved .github/ISSUE_TEMPLATE/feature_request.md

**Review by robertdumitrescu — APPROVED** [review]

**Review by yghern — APPROVED** [review]
Looks all right

**Pull request merged**